### DISCUSSION TITLE
Using Weasis with Zulu JRE

### DISCUSSION THREAD

**Discussion by ngaither-tdh** [nroduit/Weasis#567] [General]
Hello,

I cannot find any documentation regarding this, and simply installing the Zulu JRE does not work. Is Weasis compatible with any other JRE other than Java? We have removed Java completely at our organization due to the new licensing changes announced by Oracle. Our chosen replacement JRE is the Zulu JRE, we have all versions (8, 11, 17) approved and available for installation. Hoping someone can shed some light on a work around or configuration to be able to launch Weasis without Oracle Java.

Thanks!

**Comment by nroduit** [comment]
I'm not sure I fully understand the question.

If you're a user, you don't need to install Java to run Weasis. Only the JRE modules needed to run Weasis are packaged with the application using the [jpackage method](https://www.baeldung.com/jar-windows-executables). The Java version used is [Adoptium](https://adoptium.net/), see also the licenses for[ third-party libraries](https://github.com/nroduit/Weasis/blob/master/3rd-party-licenses.md).

Weasis no longer has any dependency on Oracle's Java version as of 2019, see this [post](https://groups.google.com/g/dcm4che/c/3C84sscQqkQ/m/W2nvWma7AQAJ).

**Comment by ngaither-tdh** [comment]
Hmmm, our users are presented with this screen after Java was removed from their system.
![image](https://github.com/nroduit/Weasis/assets/144953151/1a6c026f-c6f0-490d-a403-609f8171b776)


Reading through some of the articles you attached, Java is not required any longer, which would make me think this screen wouldn't show up. It might be possible our users are distributing an extremely old version of Weasis. I will reach out and see if that is the case.

I will report back the findings. In the case where we have a version newer than 3.5.3 I will seek additional help. 

Thank you!
